testbed/correctness/metrics/correctness_test_case.go

```diff
@@ -0,0 +1,103 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
It seems this functionality should be in the testbed package like traces test case. It will need to be used for correctness tests in the contrib project.

Reply:
If that is the approach then this PR is fine. I will change my review status.